### DISCUSSION TITLE
Why and When use stop instead of off?

### DISCUSSION THREAD

**Discussion by ZeFish** [music-assistant/support#3022] [Q&A] [answered]
I'm using "turn off" to my players to stop them and was wondering if it would be better using stop.
I'm using google minis. Would using stop keep the streaming open so during the next play I would not ear the starting google sound?

Sorry if I'm not clear enough.

**Comment by OzGav** [comment]
The Google “ding” is not something which MA can control.

**Reply by ZeFish** [reply]
Yeah that what I was thinking but what's the different use case between stop and off?

**Reply by OzGav** [reply]
You can’t stop from the MA UI only pause so do you mean difference between paused and off?

**Reply by ZeFish** [reply]
I mean, in the action menu while making scripts, I have choice between Media Player : Stop and Media Player : Turn Off and so far I've only used turn off but I'm wondering whats the difference between the two.

I wonder if stop does stop the music but keep the connection alive so next time I trigger a play command, Music assistant wont have to connect again since it kept it alive. But I don't know.

**Reply by OzGav** [reply]
More info about the various states are here https://developers.home-assistant.io/docs/core/entity/media-player/

**Reply by marcelveldt** [reply, marked as answer]
Yes, stop keeps the connection alive so in theory you wont hear the google ding when you start playback but I'm pretty sure that google kicks out an inactive app on its own at some point.

In the UI you can simply always press the pause button. It will automatically stop after 30 seconds.
